controller.py

```diff
@@ -192,9 +193,63 @@ def canonicalize_author_name(self):
         return make_response(author_name, HTTP_OK, {"Content-Type": "text/plain"})
 
 
-class CatalogController(object):
+class ISBNEntryMixin(object):
+
+    def make_opds_entry_from_metadata_lookups(self, identifier):
```

Comment:
This method is copied almost verbatim from URNLookupController except for two small bits below.